.github/issue_template.md

```diff
@@ -0,0 +1,7 @@
+#### Issue
```

Comment:
Let's uppercase filename and add this to `.npmignore`. We don't need this in end-up bundle.